### PR TITLE
feat: enable custom-links page

### PR DESCRIPTION
**Pull request by alfetopito**
# Description

Enable custom-links page

It seems to have been available since the start, but it was always set to `draft`
image

# Changes

- [ ] Remove `draft` flag from `custom-links` page
- [ ] Add example using `recipient`
- [ ] Removed Goerli from everywhere

### CODE REVIEW

**Comment by mfw78** [comment]
FYI, the reason why this wasn't taken out of draft is it wasn't clear where this should fit within the table of contents, as for example the widget is CoW Swap, but CoW Swap tutorials as they are at the moment are targeted at users and not integrations. Similar analogy for Custom Links, they are a feature of CoW Swap, but are targeted at integrators.

**Comment by elena-zh** [comment]
Hey @alfetopito , great, thank you!

Some not related questions (but maybe you know answers) :)
1. WDYT about adding one more example with recipient param? 
image
2. I can see that Goerly is listed as a supported network in the docs. In this [article](https://docs-git-feat-enable-custom-links-page-cowswap.vercel.app/cow-protocol/reference/contracts/core#deployments)
image
and at the bottom of the current page 
image
Should it be removed? LMK if I need to open a separate issue for this. 
cc @mfw78 

Thanks!

**Review by elena-zh — APPROVED** [review]

**Comment by alfetopito** [comment]
> FYI, the reason why this wasn't taken out of draft is it wasn't clear where this should fit within the table of contents, as for example the widget is CoW Swap, but CoW Swap tutorials as they are at the moment are targeted at users and not integrations. Similar analogy for Custom Links, they are a feature of CoW Swap, but are targeted at integrators.

Where should this go then?
We do need to have this public as multiple times people ask about this and there's no doc where to point them to.

**Comment by alfetopito** [comment]
> Hey @alfetopito , great, thank you!
> 
> Some not related questions (but maybe you know answers) :)
> 
>     1. WDYT about adding one more example with recipient param?
>        image
> 
>     2. I can see that Goerly is listed as a supported network in the docs. In this [article](https://docs-git-feat-enable-custom-links-page-cowswap.vercel.app/cow-protocol/reference/contracts/core#deployments)
>        image
>        and at the bottom of the current page
>        image
>        Should it be removed? LMK if I need to open a separate issue for this.
>        cc @mfw78
> 
> 
> Thanks!

Updated

**Comment by elena-zh** [comment]
Hey @alfetopito , nitpick!
Ethereum does not have WXDAI tokens!
image
`https://swap.cow.fi/#/1/swap/WETH/COW?buyAmount=10000&recipient=vitalik.eth `?

All the rest changes LGTM!

**Comment by mfw78** [comment]
> Where should this go then?
> We do need to have this public as multiple times people ask about this and there's no doc where to point them to.

I've got no strong opinion besides to keep things resembling some level of organisation. If tutorials are biased towards integrators, they should be grouped, and vice-versa, if they're biased towards users of products (ie. CoW Swap, CoW Explorer), they should be grouped as well.

File: docs/cow-protocol/tutorials/cow-swap/custom-links.md [hunk 1]
```diff
@@ -15,6 +14,7 @@ draft: true
 - [`https://swap.cow.fi/#/100/swap`](https://swap.cow.fi/#/100/swap) - `swap` page on Gnosis chain network
 - [`https://swap.cow.fi/#/100/limit/WXDAI/COW`](https://swap.cow.fi/#/100/limit/WXDAI/COW) - Create a `limit` order to sell `WXDAI` for `COW`
 - [`https://swap.cow.fi/#/1/swap/WETH/COW?sellAmount=100`](https://swap.cow.fi/#/1/swap/WETH/COW?sellAmount=100) - Swap `100 WETH` for `COW`
+- [`https://swap.cow.fi/#/1/swap/WXDAI/COW?buyAmount=10000&recipient=vitalik.eth`](https://swap.cow.fi/#/1/swap/WETH/COW?buyAmount=10000&recipient=vitalik.eth) - Buy on `swap` sending the outcome to a custom recipient (can be either address or ens)
```
Comment (mfw78) [inline docs/cow-protocol/tutorials/cow-swap/custom-links.md]:
```suggestion
- [`https://swap.cow.fi/#/1/swap/WETH/COW?buyAmount=10000&recipient=vitalik.eth`](https://swap.cow.fi/#/1/swap/WETH/COW?buyAmount=10000&recipient=vitalik.eth) - Buy on `swap` sending the outcome to a custom recipient (can be either address or ens)
```

**Review by mfw78 — APPROVED** [review]
Will potentially reorganise the tutorials in another PR to keep to a  user v integrators organisation strategy.

**Pull request merged**